src/_content/posts/introducing-keynote-mario-munoz.md

```diff
@@ -0,0 +1,59 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 2024-08-01 06:00:00
+cover:
+    url: /assets/img/blog/mario_munoz.jpg
+    alt: "Mario Munoz"
+layout: post
+title: "Announcing Our Keynotes: Mario Munoz"
+---
+
+### We're honored to announce our seecond keynote speaker for DjangoCon US 2024: Mario Munoz!
```

Comment:
@jcjudkins is there a reason why this is a `h3` element instead of `h2`?

```suggestion
### We're honored to announce our second keynote speaker for DjangoCon US 2024: Mario Munoz!
```